packages/cli/package.json

```diff
@@ -33,7 +33,7 @@
         "@types/lodash": "^4.17.6",
```

Comment:
Please update accessibility-insights-scan version for release.

Reply:
ok